pepdata/danafarber.py

```diff
@@ -38,16 +38,15 @@ def load_tumor(
     """
     path = cache.fetch(
         filename="tumor_epitopes.csv",
-        download_url="http://bio.dfci.harvard.edu/DFRMLI/datasets/tumor_epitopes.htm",
-        subdir="pepdata")
+        url="http://projects.met-hilab.org/DFRMLI/datasets/tumor_epitopes.htm")
     df = pd.read_csv(path, skipinitialspace=True, nrows=nrows)
-    mask = ~df.Peptide.str.contains(bad_amino_acids)
+    mask = ~df.Epitope.str.contains(bad_amino_acids)
     if peptide_length:
-        mask &= df.Peptide.str.len() == peptide_length
+        mask &= df.Epitope.str.len() == peptide_length
     if hla_type:
         mask &= df.Allele.str.contains(hla_type)
     if source_protein:
-        mask &= df.Protein.str.contains(source_protein)
+         mask &= df["Tumor antigen"].str.contains(source_protein)
```

Comment:
This feels like a dumb question, but is that whitespace before `mask` correct?

Reply:
not a dumb question at all and yes that was whitespace.  I removed and amended the commit.